presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -12,7 +12,7 @@ services:
     image: ${HADOOP_MASTER_IMAGE}
     hostname: hadoop-master
     ports:
-      - '1080:1080'
+      - '${HIVE_PROXY_PORT}:1080'
```

Comment:
I believe you have this backwards. The first number is the internal port, the second is the one you're exposing it on.
This should read:
```
- '1080:${HIVE_PROXY_PORT}'
```